torchchat/usages/openai_api.py

```diff
@@ -288,13 +289,50 @@ def __init__(self, *args, **kwargs):
                 else self.model.text_transformer_args.max_seq_length
             )
         except:
-            # can not find max_seq_length in model config, use default value
-            self.max_seq_length = 128
```

Comment:
haha now i can see where 128 comes from

Reply:
It happens to be the same as the head_dim, very tricky to trace it LOL

Reply:
yep.

Reply:
Thanks to @iseeyuan for spotting it.